src/appMain/start_external_proprietary.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+python sample_policy_manager.py --pack_port 8088 --unpack_port 8089 &
+POLICY_MANAGER=$!
```

Comment:
I think Core should still continue running since it is independent of the external policy manager. Also errors from starting the python script will be logged in the console